statute.md

```diff
@@ -0,0 +1,106 @@
+* Credentials management and sharing
+
+1.2
+
+The Statute exists to foster participation in Berlin TWC by creating a transparent and democratic decisional and organizational framework. 
+ 
+1.3
+Berlin TWC has as its primary goal the empowerment of Tech Workers in their industry and in the society at large, in solidarity with other workers. Power here is understood as the capability of collectives to shape their own companies, governments. This goal will be pursued through workers’ organization, workers’ education and the progressive liberation of technological production.
+
+
+The application and modification of this document must be driven by the principles of fairness, horizontality, inclusivity organizational scalability and safety of the members. 
+
+
+1.4 
+
+The work conducted for and under Berlin TWC should aim to be transparent to the organization itself. While a degree of autonomy is recognized to individuals and Teams for operative purposes, sharing and reporting the activities conducted must be held as an high-priority aspect of one’s own participation in Berlin TWC. 
+
+1.5
+
+Modifications to this document can be proposed to the Assembly and can be passed with a majority of (3/4)+1.
+
+# 2 
+
+2.1
+
+A Berlin TWC Assembly is a meeting that is defined by a date, a location and an agenda produced by the Coordination Team (defined later). 
+
+
+2.2
+
+A member of Berlin TWC is a person that has been ratified during an Assembly.
+
+2.2.1
+
+The requirements to apply for membership are the following:
+
+* participation in at least 3 Berlin TWC events within the last six months
+* a sponsorship by a current member of Berlin TWC
+
+The sponsor can present your application and guarantee that you have participated in the past. 
+
+Any member in the Assembly can raise a case against the applicant. If one or more cases are raised, the Assembly must vote to confirm or reject the applicant. A rejected applicant can apply again after 3 months.
+
+An assembly can revoke the membership to a member that acted against the interests and principles of Berlin TWC. To revoke a membership, a member must raise a case that will be discussed in the Assembly to revoke or confirm the membership.
+
+2.2.2 
+
+Members that physically participate in the Assembly have the right to vote. A simple majority is sufficient to approve any decision, unless specified otherwise. The vote is collected by a show of hands.
+
+2.3
+
+Every proposal must be submitted to the Coordination Team at least 3 days before the Assembly. A proposal should be in the form of a binding statement that can be either approved or rejected. If the statement binds a specific Team, it must be said explicitly. 
+
+3
+
+A Team is a self-organised group of Berlin TWC members. A member can belong to zero or more teams. Every Team must have at least one Credentials' holder (defined later) for the tools necessary for the team to perform their tasks.
```

Comment:
Can non members be involved with a team? I would argue yes, and explicitly add language. `Members and non-members can belong to zero or more teams. Some Teams can restrict membership, for example handling sensitive information`